python/cudf_polars/cudf_polars/dsl/expressions/base.py

```diff
@@ -131,30 +131,26 @@ def evaluate(
         except KeyError:
             return self.do_evaluate(df, context=context, mapping=mapping)
 
-    def collect_agg(self, *, depth: int) -> AggInfo:
+    @property
+    def agg_request(self) -> plc.aggregation.Aggregation:
         """
-        Collect information about aggregations in groupbys.
-
-        Parameters
-        ----------
-        depth
-            The depth of aggregating (reduction or sampling)
-            expressions we are currently at.
+        The aggregation for this expression in a grouped aggregation.
 
         Returns
         -------
-        Aggregation info describing the expression to aggregate in the
-        groupby.
+        Aggregation request. Default is to collect the expression.
```

Comment:
>Default is to collect the expression.

Just to check my understanding: The default return is `plc.aggregation.collect_list()`. Therefore, "collect the expression" means we are collecting the list of values produced by the expression. Correct?

Reply:
Yes.